model/Mastermind.php

```diff
@@ -183,4 +216,50 @@ public function decrementRemainingAttempts(): void
         $this->remainingAttempts--;
     }
 
+    /**
+     * @param Combination $proposition
+     * @throws Exception Si la partie n'existe pas ou qu'elle est terminée
+     */
+    public function play(Combination $proposition): void
+    {
+        if ($this->remainingAttempts <= 0) {
+            throw new Exception('Partie non créée');
+        } else {
+            $this->addProposition($proposition);
+            $comparator = new CombiComparator($this->getSolution());
+            $this->addCompareResult($comparator->compare($proposition));
```

Comment:
Pareil ici, tu peux directement utiliser `$this->compareResults[] = $comparator->compare($proposition);`